app/cdap/components/WranglerGrid/SelectColumnPanel/styles.ts

```diff
@@ -0,0 +1,240 @@
+import {
```

Comment:
Missing license

Reply:
added license

Reply:
This makes me think that something got messed up in the commit squashing - I'll give it another full review

Reply:
sure Sean
Thank you

Reply:
@seanfeld plz let us know an ETA for completing your review.
I will continue work on this PR if any comments are added.
Also, please suggest if we have to do any changes to this PR so we can expedite the process.

Thank you.